xmodule/capa/safe_exec/safe_exec.py

```diff
@@ -178,6 +179,21 @@ def safe_exec(
         else:
             emsg = None
 
+        # Run the code in both the remote codejail service as well as the local codejail
+        # when in darklaunch mode.
+        if is_codejail_in_darklaunch():
+            data = {
+                "code": code_prolog + LAZY_IMPORTS + code,
+                "globals_dict": globals_dict,
+                "python_path": python_path,
+                "limit_overrides_context": limit_overrides_context,
+                "slug": slug,
+                "unsafely": unsafely,
+                "extra_files": extra_files,
+            }
+
+            get_remote_exec(data)
```

Comment:
It feels like you should be able to unit test the new code with various mocks of `codejail_safe_exec` and `get_remote_exec`, including mocking return values and ensuring that the right results get returned when both are called.